tests/Api/UsersTest.php

```diff
@@ -83,7 +83,7 @@ public function testLoginEmptyPassword(): void
     public function testGetUserByEmailSuccess(): void
     {
         $userEmail = 'user@example.com';
-        $this->configureMessage('GET', '/users/email/'.$userEmail);
+        $this->configureMessage('GET', '/users/email/' . $userEmail);
```

Comment:
why those changes? :thinking:

Reply:
It was was auto formatted by default php code format settings in phpStorm.